docs/editor/artificial-intelligence.md

```diff
@@ -0,0 +1,95 @@
+---
+Order: 7
+Area: editor
+TOCTitle: Refactoring
```

Comment:
What would you like to show in the TOC? Maybe "AI Tools", or "Coding with AI" or do you want to call out CoPilot directly?

And for the ContentId, you can just create a new GUID. The Create GUID extensions (nwallace.createguid) works well.

Reply:
Ah good catch! Updated to "AI Tools" and created a GUID.